### DISCUSSION TITLE
Having Issue in Setup Nodejs server with https domain

### DISCUSSION THREAD

**Discussion by BhartiCogniter** [community/community#28138] [Other Feature Feedback, Questions, & Ideas]
Hii 

As we are creating one to one chat application using webrtc and nodejs server. Everything is working fine on localhost with http and https both. And when we are deploying it to live server https domain it is not connecting through the project but connecting well when we make it http as our project setup is on https domain so we need to work with https only....
Please advise. It will be grateful.
Thanks in advance!

**Comment by mshrnv** [comment]
make sure you are in the right way

**Comment by Watchinggaysex** [comment]
We are also provide crack software. It has a wonderful feature. This is a good job. It is easy to use. 
https://azanpc.com/itop-vpn-crack-download-latest-version/

**Reply by LUIS9799** [reply]
Malware.

**Comment by LUIS9799** [comment]
You need to setup SSL to go https, when you deploy locally it may receive https because of self-signed cert, however if you want to deploy in other server you should use [OpenSSL](https://www.openssl.org/), or [Let's Encrypt](https://letsencrypt.org/) in order to setup a key for getting ssl requests, if your DNS (Domain) gets forced SSL (https) you can deactivated using a DNS Manager as [Cloudflare](https://cloudflare.com/) if your server have cpanel simply add keys into SSL option, if not, you can use OpenSSL self-signed CLI option with ```openssl req -new > cert.csr
openssl rsa -in privkey.pem -out key.pem
openssl x509 -in cert.csr -out cert.pem -req -signkey key.pem -days 1001
cat key.pem>>cert.pem```

I hope it helped, let me know if it works!!

**Comment by LUIS9799** [comment]
Also you're not on the right section, you should ask this kind of questions in other topic, this is **Profile **topic, make sure you are on the right place!
